go/vt/schemadiff/table.go

```diff
@@ -445,6 +445,18 @@ type CreateTableEntity struct {
 	Env *Environment
 }
 
+func NewCreateTableEntityFromSQL(env *Environment, sql string) (*CreateTableEntity, error) {
```

Comment:
@shlomi-noach Do we want to have a symmetric function for views here?

Reply:
Added + testing.